src/main/java/org/folio/marccat/dao/persistence/CorrelationKey.java

```diff
@@ -89,12 +77,13 @@ private void setMarcSecondIndicator(char c) {
   }
 
   public CorrelationKey changeSkipInFilingIndicator(int skip) {
-    if (marcFirstIndicator == 'S') return changeFirstIndicator(Integer.toString(skip).charAt(0));
-    if (marcSecondIndicator == 'S') return changeSecondIndicator(Integer.toString(skip).charAt(0));
+    if (marcFirstIndicator == 'S')
+      return changeFirstIndicator(Integer.toString(skip).charAt(0));
```

Comment:
Please be consistent with syntax/format... here you omit the curly brackets for a single line if block, but below you include them.

Reply:
Thanks a lot. I have changed it.